protocol/messenger_edit_message_test.go

```diff
@@ -90,6 +90,88 @@ func (s *MessengerEditMessageSuite) TestEditMessage() {
 	s.Require().Equal(ErrInvalidEditOrDeleteAuthor, err)
 }
 
+func (s *MessengerEditMessageSuite) TestEditImageMessage() {
+	theirMessenger := s.newMessenger()
+	defer TearDownMessenger(&s.Suite, theirMessenger)
+
+	theirChat := CreateOneToOneChat("Their 1TO1", &s.privateKey.PublicKey, s.m.transport)
+	err := theirMessenger.SaveChat(theirChat)
+	s.Require().NoError(err)
+
+	ourChat := CreateOneToOneChat("Our 1TO1", &theirMessenger.identity.PublicKey, s.m.transport)
+	err = s.m.SaveChat(ourChat)
+	s.Require().NoError(err)
+
+	const messageCount = 1
+	var album []*common.Message
+
+	for i := 0; i < messageCount; i++ {
```

Comment:
`const messageCount = 1` :eyes:

Reply:
lol yes. I didn't know how to send only one image so I copied the code from the album test which sends multiple images.

The test becomes way more complex for no reason when using more than one image, so I decided to just put `messageCount = 1` :sweat_smile: